sky/backends/backend_utils.py

```diff
@@ -481,6 +482,9 @@ def _add_multinode_config(
         sky_autogen_comment = ('# Added by sky (use `sky stop/down '
                                f'{cluster_name}` to remove)')
 
+        # Ensures stableness of the aliases worker-<i>.
+        worker_ips = list(sorted(worker_ips))
```

Comment:
This means `worker<i>` is sorted by public IPs, right? (Also, we should add "public" to the var name to be clearer.)

Reply:
Yup, we sort based on public IPs.